### PR TITLE
Added Fetkovich Test Case

### PR DESCRIPTION
**Pull request by WesselDeZeeuw**
This is the first test case for the Fetkovich Aquifer. 
It is a 5 x 20 2D model, with a multiconnected aquifer.

### CODE REVIEW

**Comment by alfbr** [comment]
@WesselDeZeeuw please inform when you believe this is ready for review/merge.

**Comment by WesselDeZeeuw** [comment]
> @WesselDeZeeuw please inform when you believe this is ready for review/merge.

Yes, I believe it's ready for review/merge.

**Comment by alfbr** [comment]
Good. Thanks for the contribution, good test cases are invaluable! As far as I can see, the test data set is ready to go in. Reference solutions can be auto generated by the test system. Hence, for review purposes it is actually better to leave them all out of the pull request. That means that you can keep the top level DATA file, and delete all the other files. Please also squash the commits afterwards so that we avoid having all the binaries in the git history.

**Comment by GitPaean** [comment]
In the reviewing process for PR https://github.com/OPM/opm-simulators/pull/1642 , we added some ACTNUM to the DATA file to remove some grid cells to make the PR aquifer related implementation more general. Maybe we can incorporate that parts to this example. @alfbr , @WesselDeZeeuw . 

In anyway, it will protect more implementation through regression test.

**Comment by WesselDeZeeuw** [comment]
> Good. Thanks for the contribution, good test cases are invaluable! As far as I can see, the test data set is ready to go in. Reference solutions can be auto generated by the test system. Hence, for review purposes it is actually better to leave them all out of the pull request. That means that you can keep the top level DATA file, and delete all the other files. Please also squash the commits afterwards so that we avoid having all the binaries in the git history.

Done so! :-)

File: aquifer-fetkovich/eclipse-simulation/2D_FETKOVICHAQUIFER.DATA [hunk 1]
```diff
@@ -0,0 +1,648 @@
+-- This reservoir simulation deck is made available under the Open Database
```
Comment (GitPaean) [inline aquifer-fetkovich/eclipse-simulation/2D_FETKOVICHAQUIFER.DATA]: This file is not needed.

File: aquifer-fetkovich/opm-simulation-reference/flow/2D_FETKOVICHAQUIFER.DATA [hunk 2]
```diff
@@ -0,0 +1,648 @@
+-- This reservoir simulation deck is made available under the Open Database
```
Comment (GitPaean) [inline aquifer-fetkovich/opm-simulation-reference/flow/2D_FETKOVICHAQUIFER.DATA]: This file is not needed either.

File: aquifer-fetkovich/eclipse-simulation/2D_FETKOVICHAQUIFER.DATA (hunk 1 above)
Comment (WesselDeZeeuw) [reply aquifer-fetkovich/eclipse-simulation/2D_FETKOVICHAQUIFER.DATA]: Ok! Should I keep the directories? Or can I remove these too?

**Comment by alfbr** [comment]
Excellent, as @GitPaean explains, only the top level DATA file is needed. Please remove the two others and squash commits again.

File: aquifer-fetkovich/eclipse-simulation/2D_FETKOVICHAQUIFER.DATA (hunk 1 above)
Comment (GitPaean) [reply aquifer-fetkovich/eclipse-simulation/2D_FETKOVICHAQUIFER.DATA]: I think you do not need to keep the directories unless you need to have files there.

Comment (alfbr) [reply aquifer-fetkovich/eclipse-simulation/2D_FETKOVICHAQUIFER.DATA]: Actually, git does not care about directories, so you do not need to do anything with them. Git only tracks files

**Comment by alfbr** [comment]
That wraps it up nicely, thanks for your patience. Merging!

**Pull request merged**